src/components/header/examples/header.tsx

```diff
@@ -0,0 +1,48 @@
+import { Component, h, State } from '@stencil/core';
+import { Action } from '@limetech/lime-elements';
+
+@Component({
+    tag: 'limel-example-header',
+    shadow: true,
+})
+export class HeaderExample {
+    @State()
+    private actions = [
+        {
+            id: '1',
+            icon: 'refresh',
+            label: 'Refresh',
+        },
+        {
+            id: '2',
+            icon: 'delete',
+            label: 'Delete',
+            disabled: true,
+        },
+        {
+            id: '3',
+            icon: 'edit',
+            label: 'Edit',
+        },
+    ];
+
+    constructor() {
+        this.handleAction = this.handleAction.bind(this);
```

Comment:
No necessary to bind this for this example

Reply:
Correct, since we don't access `this` inside `handleAction`. But since any real world `handleAction` would almost certainly access `this`, and this is an example that people will look at to learn, maybe we should still bind it… 🤔

The best argument I can see against it is that it might be confusing to some if we use `bind` where we don't need it.

Just my 2¢, I'll leave the decision to you people 😊

Reply:
Decided to leave this here just for this example.

Reply:
What I wanted to say is it might be confusing to some if we use bind where we don't need it.

Reply:
Yeah that make sense, I removed this binding in the example.